tests/standalone/query-errors.phpt

```diff
@@ -1,37 +0,0 @@
---TEST--
-MongoDB\Driver\Query: Invalid types
```

Comment:
`query-ctor_error-002.phpt` makes this test redundant, as it covers these types and more.